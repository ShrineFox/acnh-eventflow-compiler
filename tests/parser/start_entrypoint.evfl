flow Test():
entrypoint EP1:
    TestActor1.Action0()

