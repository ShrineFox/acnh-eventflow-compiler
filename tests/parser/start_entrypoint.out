RootNode[name=EP1, vardefs=[], out_edges=[Event0]]
RootNode[name=Test, vardefs=[], out_edges=[Event0]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[]]

